docs/resources/samples/add-excel-comments.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Add comments in Excel'
+description: 'Learn how to use Office Scripts to add comments in a worksheet.'
+ms.date: 03/18/2021
+localization_priority: Normal
+---
+
+# Add comments in Excel
+
+This sample shows how to add comments to a cell with at-mention of a contact.
+
+## Example scenario
+
+* Team lead maintains shift schedule information. Team lead assigns employee ID to shift record.
+* Team lead wishes to send email reminder by adding a comment with @mention of the employee by cross referencing the employee email address along with a custom message from the sheet.
+* Employee can then address and respond to the comment.
```

Comment:
Small note to look at the grammar of these bullets. I think they could be made into complete sentences.